website/landingpage/dev/src/assets/scss/_general.scss

```diff
@@ -148,16 +148,6 @@ section,
         margin-bottom: -200px;
         padding-bottom: calc(200px + 100px);
       }
-
-      &.has-big-bottom-margin {
```

Comment:
please don't remove this class from HTML and styles for it, otherwise, there is a gap on desktop resolution
<img width="1200" alt="desktop" src="https://user-images.githubusercontent.com/46446373/81404786-97d80680-9136-11ea-8fc7-8c2072cf14f7.png">